class/Defaults/MergeTag/Comment/CommentContent.php

```diff
@@ -44,6 +44,7 @@ public function __construct( $params = array() ) {
 				'resolver'    => function( $trigger ) {
 					return $trigger->{ $this->property_name }->comment_content;
 				},
+				'group' => sprintf( __( 'Comment data', 'notification' ) ),
```

Comment:
Hmm, a bit of redundancy with this name I think just `Comment` instead of `Comment data` would be sufficient